package.json

```diff
@@ -10,6 +10,8 @@
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.6.0",
     "history": "^4.7.2",
+    "jest-localstorage-mock": "^2.3.0",
```

Comment:
What is the use of this?

Reply:
@dondrzzy  . I was trying to run tests for an action which depended on input from local storage. Got another way of testing it but forgot to uninstall the package.
The package has been uninstalled. Thanks for spotting that.